autofill/autofill-api/src/main/java/com/duckduckgo/autofill/api/feature/AutofillFeatures.kt

```diff
@@ -14,9 +14,18 @@
  * limitations under the License.
  */
 
-package com.duckduckgo.autofill.api
+package com.duckduckgo.autofill.api.feature
 
-/** List of [AutofillFeatureName] that belong to the Autofill feature */
 enum class AutofillFeatureName(val value: String) {
     Autofill("autofill"),
 }
+
+enum class AutofillSubfeatureName(override val value: String) : AutofillSubfeature {
```

Comment:
Not used publicly but they are used from both `autofill-impl` and `autofill-store` modules, so i think leaving it defined in `autofill-api` is ok here?

Reply:
Public means out of autofill so if there are no usages out of autofill I would move that. Worse case scenario at some point we need this public and we copy back to the api but since your public api is the capabilities checker that seems unlikely.

Reply:
Discussed, we'll leave in api because it's required in the store.

Reply:
Decision from conversation elsewhere: leaving it as is since it is used by `impl` and `store` (Could make the store use strings instead of the stronger typing but not worth doing)